tests/unit/conftest.py

```diff
@@ -253,22 +253,24 @@ def harness_container_fixture(harness: Harness, container: Container) -> Harness
     return HarnessWithContainer(harness=harness, container=container)
 
 
-@pytest.fixture(scope="function", name="raise_exception")
-def raise_exception_fixture():
+@pytest.fixture(scope="function", name="raise_exception_mock")
+def raise_exception_mock_fixture():
```

Comment:
It is quite easy to create a mock object that raises an exception when it is called:

```
>>> from unittest import mock
>>> test = mock.MagicMock(side_effect=Exception)
>>> test()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.10/unittest/mock.py", line 1114, in __call__
    return self._mock_call(*args, **kwargs)
  File "/usr/lib/python3.10/unittest/mock.py", line 1118, in _mock_call
    return self._execute_mock_call(*args, **kwargs)
  File "/usr/lib/python3.10/unittest/mock.py", line 1173, in _execute_mock_call
    raise effect
Exception
```

Is it worth creating a fixture given the mock creation is a single line of code?

Reply:
Fixed, thank you!
Also fixed consistency in imports